Core/src/Geometry/MultiWireVolumeBuilder.cpp

```diff
@@ -0,0 +1,119 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "Acts/Geometry/MultiWireVolumeBuilder.hpp"
+
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/NavigationPolicyFactory.hpp"
+#include "Acts/Geometry/TrapezoidPortalShell.hpp"
+#include "Acts/Geometry/TrapezoidVolumeBounds.hpp"
+#include "Acts/Navigation/InternalNavigation.hpp"
+#include "Acts/Navigation/MultiLayerNavigationPolicy.hpp"
+#include "Acts/Navigation/TryAllNavigationPolicy.hpp"
+#include "Acts/Surfaces/SurfaceArray.hpp"
+#include "Acts/Utilities/Helpers.hpp"
+#include "Acts/Utilities/StringHelpers.hpp"
+
+Acts::Experimental::MultiWireVolumeBuilder::MultiWireVolumeBuilder(
```

Comment:
if you do `namespace Acts::Examples {...}` like in the header you can safe a lot of characters by dropping all `Acts::` and `Acts::Experimental::`